Removed everything associated to mockserver in browser

KateN87 opened the pull request:
- uninstalled msw
- removed msw workerDirectory from package.json
- deleted mockserviceWorker.js -deleted browser.js, handlers.jsx, mockUsers.json from mock folder -deleted mock folder
-moved mockUsers to src
-removed import mock browser from main

KateN87 committed:
Removed everything associated to mockserver in browser

- uninstalled msw
- removed msw workerDirectory from package.json
- deleted mockserviceWorker.js
-deleted browser.js, handlers.jsx, mockUsers.json from mock folder
-deleted mock folder
-moved mockUsers to src
-removed import mock browser from main

patilnandini1811 commented:
Everything looks correct, Kate. I see you've uninstalled MSW and removed all related configurations and files. Additionally, moving mockUsers.json to the src directory seems appropriate. Just make sure that all references to these files and configurations are also updated in the code. As I made chnages in Login files!

patilnandini1811 commented:
Everything looks correct, Kate. I see you've uninstalled MSW and removed all related configurations and files. Additionally, moving mockUsers.json to the src directory seems appropriate. Just make sure that all references to these files and configurations are also updated in the code. As I made chnages in Login files!

KateN87 commented:
> Everything looks correct, Kate. I see you've uninstalled MSW and removed all related configurations and files. Additionally, moving mockUsers.json to the src directory seems appropriate. Just make sure that all references to these files and configurations are also updated in the code. As I made chnages in Login files!

Thank you! Did you see if I missed any imports? Cause I tried to find everyone and everything seems to be working on my end.
